plugins/secretstores/jwt/jwt.go

```diff
@@ -0,0 +1,141 @@
+package jwt
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/secretstores"
+	"io"
+	"net/http"
+	"net/url"
```

Comment:
Please group the imports into *built-in libraries*, *3rd-party libraries* and *telegraf internal libraries* each group in alphabetical order and separated by an empty line
```suggestion
	"bytes"
	"encoding/json"
	"errors"
	"fmt"
	"io"
	"net/http"
	"net/url"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/plugins/secretstores"
```